packages/lsp-tools/src/doenet-to-markdown/index.ts

```diff
@@ -0,0 +1,254 @@
+                            ),
+                        },
+                    ];
+                case "tag":
+                    return [
+                        {
+                            type: "inlineCode",
+                            value: `<${textContent(node)}>`,
+                        },
+                    ];
+                case "c":
+                    return [
+                        {
+                            type: "inlineCode",
+                            value: textContent(node),
+                        },
+                    ];
+                case "ol":
+                    return [
+                        {
+                            type: "list",
+                            ordered: true,
+                            children: mapChildren(node.children).filter(
+                                (n) => n.type === "listItem",
+                            ) as MdastListItem[],
+                        },
+                    ];
+                case "li":
+                    return [
+                        {
+                            type: "listItem",
+                            children: mapChildren(
+                                node.children,
+                            ) as MdastBlockContent[],
+                        },
+                    ];
+                case "m":
+                case "math":
+                    return [
+                        {
+                            type: "text",
+                            value: `$${textContent(node)}$`,
+                        },
+                    ];
+                case "p":
+                    return [
+                        {
+                            type: "paragraph",
+                            children: mapChildren(
+                                node.children,
+                            ) as MdastPhrasingContent[],
+                        },
+                    ];
+                case "title": {
+                    const depth = sum(
+                        sourceObj
+                            .getParents(node as DastNodesV6)
+                            .map((n) =>
+                                n.type === "element" &&
+                                DIVISIONS.includes(n.name)
+                                    ? 1
+                                    : 0,
+                            ),
+                    );
+                    return [
+                        {
+                            type: "heading",
+                            depth: clampToHeadingNumber(depth),
+                            children: mapChildren(
+                                node.children,
+                            ) as MdastPhrasingContent[],
+                        },
+                    ];
+                }
+                case "section":
+                    // Directly pass through. Everything will be handled by the `<title>` element
+                    return node.children.flatMap((n) =>
```

Comment:
Or just `return mapChildren(node.children);` to emphasize it is the same as others.